RadixWallet/Clients/ProfileStore/ProfileStore.swift

```diff
@@ -583,35 +591,35 @@ extension ProfileStore {
 
 	private static func _newProfileAndBDFSMnemonic(
 		deviceInfo creatingDevice: DeviceInfo
-	) throws -> (profile: Profile, bdfsMnemonic: PrivateHDFactorSource) {
+	) throws -> (profile: Profile, bdfsMnemonic: PrivateHierarchicalDeterministicFactorSource) {
 		@Dependency(\.uuid) var uuid
 		@Dependency(\.date) var date
 		@Dependency(\.mnemonicClient) var mnemonicClient
 
 		let profileID = uuid()
-		let header = ProfileSnapshot.Header(
+		let header = Profile.Header(
+			snapshotVersion: .v100,
+			id: profileID,
 			creatingDevice: creatingDevice,
 			lastUsedOnDevice: creatingDevice,
-			id: profileID,
 			lastModified: date.now,
-			contentHint: .init()
+			contentHint: .init(
+				numberOfAccountsOnAllNetworksInTotal: 0,
+				numberOfPersonasOnAllNetworksInTotal: 0,
+				numberOfNetworks: 0
+			)
 		)
 
 		let mnemonic = try MnemonicWithPassphrase(
 			mnemonic: mnemonicClient.generate(
-				BIP39.WordCount.twentyFour,
-				BIP39.Language.english
-			)
+				BIP39WordCount.twentyFour,
+				BIP39Language.english
+			), passphrase: ""
```

Comment:
FIXME: make passphrase optional

Reply:
not blocking merge, but can try to tackle on Monday if there is time